doc/Contribution.md

```diff
@@ -0,0 +1,118 @@
+# Contribution Guidelines
+==================
+## Overview
+Being an Open Source project, everyone can contribute, provided that you respect the following points:
+* Before contributing any code, the author must make sure all the tests work (see below how to launch the tests).
```

Comment:
of course, the developer has to run his tests manually, I agree that Travis
is double check

2016-05-05 13:11 GMT+02:00 Daniel Moran <notifications@github.com>:

> In doc/Contribution.md
> <https://github.com/telefonicaid/iotagent-node-lib/pull/316#discussion_r62174488>
> :
>
> > @@ -0,0 +1,118 @@
> > +# Contribution Guidelines
> > +==================
> > +## Overview
> > +Being an Open Source project, everyone can contribute, provided that you respect the following points:
> > +* Before contributing any code, the author must make sure all the tests work (see below how to launch the tests).
>
> It's usually configured in all our repositories, but to use Travis to
> check your changes is quite slow, as you have to create the PR and wait for
> it to give you the red or green flag. It's easier for the developer to test
> that locally, so no nasty surprises will arise once the PR has been
> created. In that sense, Travis is more like a double-check for the PR
> reviewers than for the developer itself, IMHO.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/telefonicaid/iotagent-node-lib/pull/316/files/95bca228a945508e50b4c1b655fcd32969fb486b#r62174488>
>